### PR TITLE
chore: Use operations instead of actions and functions

### PR DESCRIPTION
**Pull request by marikaner**
Deprecate direct function and action acces in v2.
Related to SAP/cloud-sdk-backlog#941.

- [x] I know which base branch I chose for this PR, as the default branch is `v2-main` now, which is not for v3 development.
- [x] If my change will be merged into the `main` branch (for v3), I've updated (V3-Upgrade-Guide.md)[./V3-Upgrade-Guide.md] in case my change has any implications for users updating to SDK v3

### CODE REVIEW

File: .changeset/polite-bags-brush.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-cloud-sdk/generator': minor
+---
+
+[Compatibility Note] Direct access of action and function imports from generated services is deprecated. Use the more `operations` instead.
```
Comment (fwilhe) [inline .changeset/polite-bags-brush.md:5]: > Use the more `operations` instead

I don't understand this instruction. Can you rephrase this a little?

File: test-packages/e2e-tests/test/function.spec.ts [hunk 2]
```diff
@@ -1,14 +1,18 @@
-import {
-  returnSapCloudSdk,
-  concatStrings,
-  getByKey,
-  getAll,
-  returnCollection
-} from '@sap-cloud-sdk/test-services-e2e/v4/test-service/function-imports';
+// import {
```
Comment (fwilhe) [inline test-packages/e2e-tests/test/function.spec.ts:1]: is this supposed to be commented?

**Review by fwilhe — APPROVED** [review]
looks good to me

File: test-packages/e2e-tests/test/function.spec.ts [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
+//   returnSapCloudSdk,
+//   concatStrings,
+//   getByKey,
+//   getAll,
+//   returnCollection
+// } from '@sap-cloud-sdk/test-services-e2e/v4/test-service/function-imports';
+
```
Comment (marikaner) [inline test-packages/e2e-tests/test/function.spec.ts:8]:
```suggestion
```
